test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
I was under the impression that this is executing the function in the first arg with the rest of the args passed into the function. On that note, I'm seeing now that the array containing `context` now includes `this.globals.rules || ['section508']`. By my initial understanding of how `executeAsync` is working, should the latter be separated out into its own array? In other words, aren't those two separate arguments?